boot/bootdata/hivesys.inf

```diff
@@ -1394,7 +1394,7 @@ HKLM,"SYSTEM\CurrentControlSet\Control\ServiceGroupOrder","List",0x00010000, \
  "System Bus Extender", \
  "SCSI Miniport", \
  "Port", \
- "Primary Disk", \
+ "Primary disk", \
```

Comment:
This is just wrong, and unrelated.
Why are you mixing this into a PR about the sfloppy driver??

Reply:
Indeed, sorry, reverted that one.
Because floppy_new has that mismatch...

Reply:
The right writing is `Primary Disk`. Please fill a separate PR with that

Reply:
Moved to #3516...